docs/_config.yml

```diff
@@ -0,0 +1,73 @@
+# Welcome to Jekyll!
+#
+# This config file is meant for settings that affect your whole blog, values
+# which you are expected to set up once and rarely edit after that. If you find
+# yourself editing this file very often, consider using Jekyll's data files
+# feature for the data you need to update frequently.
+#
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# 'bundle exec jekyll serve'. If you change this file, please restart the server process.
+#
+# If you need help with YAML syntax, here are some quick references for you:
+# https://learn-the-web.algonquindesign.ca/topics/markdown-yaml-cheat-sheet/#yaml
+# https://learnxinyminutes.com/docs/yaml/
+#
+# Site settings
+# These are used to personalize your new site. If you look in the HTML files,
+# you will see them accessed via {{ site.title }}, {{ site.email }}, and so on.
+# You can create any custom variable you would like, and they will be accessible
+# in the templates via {{ site.myvariable }}.
+
+title: "Algo Builder"
+name: "Algo Builder"
+baseurl: "/algo-builder" # the subpath of your site, e.g. /blog
+url: "https://scale-it.github.io" # the base hostname & protocol for your site, e.g. http://example.com
+repository: "scale-it.github.io/algo-builder"
+
+minimal_mistakes_skin: "air" # "air", "aqua", "contrast", "dark", "dirt", "neon", "mint", "plum" "sunrise"
+
+# Build settings
+markdown: kramdown # markdown processor
+remote_theme: "mmistakes/minimal-mistakes@4.22.0"
+
+plugins:
+  - jekyll-feed
```

Comment:
is this plugin needed?

Reply:
No, it was added by default. I don't think we can remove it - [https://github.com/github/pages-gem/issues/627](https://github.com/github/pages-gem/issues/627)
For now i have commented it, with link to the open issue